src/lightning/pytorch/utilities/data/get_index.py

```diff
@@ -0,0 +1,132 @@
+import math
+import os
+
+
+def get_index(s3_connection_path: str, index_file_path: str) -> bool:
```

Comment:
why `get_index` function is named the same way as `get_index` module, this is bad pattern for imports